src/hooks/use-add-site.ts

```diff
@@ -55,7 +56,19 @@ export function useAddSite() {
 	const handleAddSiteClick = useCallback( async () => {
 		try {
 			const path = sitePath ? sitePath : proposedSitePath;
-			const newSite = await createSite( path, siteName ?? '', !! fileForImport );
+			let phpVersion = '';
+
+			if ( fileForImport ) {
+				const backupFile: BackupArchiveInfo = {
+					type: fileForImport.type,
+					path: fileForImport.path,
+				};
+
+				const meta = await getIpcApi().getMetaFromBackup( backupFile );
+				phpVersion = meta.phpVersion;
+			}
+
+			const newSite = await createSite( path, siteName ?? '', !! fileForImport, phpVersion );
```

Comment:
@sejas I wonder if we could rely on the import logic to update the PHP version after the site is created, similarly to how it works when importing for an already-created site. This way we avoid the need to extract the metafile from the import before running the import process, which would simplify the logic. WDYT?

Reply:
I also noticed that this implies extracting the backup two times which might increase significantly the duration of the creation/import process.

Additionally, I observed that after adding a site from a big import file, there's a delay when the loading screen is shown. If we follow this approach, we'd need to present the loading screen during this process.


https://github.com/user-attachments/assets/4b5e3f34-5399-4e45-9103-983cf7ad93d8

Reply:
It doesn't sound optimal to unpack the file two times during the import. I think we should fix it before we merge.